File: RAM and CPU consumption.go
package main

import (
	"fmt"
	"math/rand"
	"sort"
	"strings"
	"time"
)

func cpuHog() {
	for {
		randString := randomString(100000)
		sortedString := sortString(randString)
		_ = sortedString
	}
}
func randomString(n int) string {
	chars := "abcdefghijklmnopqrstuvwxyz"
	result := make([]byte, n)
	for i := range result {
		result[i] = chars[rand.Intn(len(chars))]
	}
	return string(result)
}
func sortString(s string) string {
	arr := strings.Split(s, "")
	sort.Strings(arr)
	return strings.Join(arr, "")
}
func memoryHog() {
	var memoryList [][]byte
	for {
		memoryList = append(memoryList, make([]byte, 10*1024*1024))
	}
}
func main() {
	for i := 0; i < 4; i++ {
		go cpuHog()
	}
	for i := 0; i < 2; i++ {
		go memoryHog()
	}
	select {}
}

File: RAM and CPU consumption.py
import threading
import time
import random
import string
def cpu_hog():
    while True:
        random_string = ''.join(random.choices(string.ascii_lowercase, k=100000))
        sorted_string = ''.join(sorted(random_string))
def memory_hog():
    memory_list = []
    while True:
        memory_list.append(' ' * 10**7)

def main():
    cpu_threads = []
    for _ in range(4):
        t = threading.Thread(target=cpu_hog)
        cpu_threads.append(t)
        t.start()
    memory_threads = []
    for _ in range(2):
        t = threading.Thread(target=memory_hog)
        memory_threads.append(t)
        t.start()
    while True:
        time.sleep(1)
if __name__ == "__main__":
    main()
